examples/tutorials/high_throughput_workgraph.ipynb

```diff
@@ -153,21 +153,12 @@
     "\n",
     "wg = get_ht_workgraph(\n",
     "    calc=SinglepointCalc,\n",
-    "    folder=Path(\"../../tests/workflows/structures/\"),\n",
+    "    folder=Path(\"/home/mtr46585/aiida-mlip/tests/workflows/structures/\"),\n",
     "    calc_inputs=inputs,\n",
```

Comment:
I know you didn't change this, but I think other changes mean `inputs` needs to be changed before this. I get an error now.

The same inputs as singlepoint, without `properties` should be fine.